Bigtable/tests/Unit/TableTest.php

```diff
@@ -133,13 +143,18 @@ public function testMutateRowsOptionalConfiguration()
                 $this->arrayAsGenerator([])
             );
         $options = [
-            'key1' => 'value1'
+            'transportOptions' => [
+                'key1' => 'value1'
+            ]
         ];
-        $this->bigtableClient->mutateRows(self::TABLE_NAME, $this->entries, $this->options + $options)
-            ->shouldBeCalled()
-            ->willReturn(
-                $this->serverStream->reveal()
-            );
+        $this->bigtableClient->mutateRows(
+            Argument::type(MutateRowsRequest::class),
+            Argument::type('array')
+        )->shouldBeCalled()
```

Comment:
So it seems like we should change this to 
```suggestion
        $this->bigtableClient->mutateRows(
            Argument::type(MutateRowsRequest::class),
            $this->options + $options
        )->shouldBeCalled()
```

This will test that the custom options we expect will show up.

I see that you've changed it from `key1` => `value1` to `transportOptions`, to make sure they're valid call options. That works for me!